go.mod

```diff
@@ -171,3 +171,5 @@ replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.14.0+incompatible
 
 replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+
+go 1.13
```

Comment:
we have not upgraded to Go 1.13 yet. We want to do this in a dedicated move. This change would force everyone onto 1.13

Reply:
@sladyn98 it would be great if we can simply remove this file changes during adding commit.